stories/Index.stories.js

```diff
@@ -0,0 +1,76 @@
+import { storiesOf } from '@storybook/vue';
+import algoliasearch from 'algoliasearch';
+
+storiesOf('ais-index', module)
+  .add('one search box, two indices', () => ({
```

Comment:
In other flavors, we usually call the first story "default" to link it more easily in the documentation.

```suggestion
  .add('default', () => ({
```